src/models/closeButton.js

```diff
@@ -155,6 +151,40 @@ const closeButtonModel = {coeffs: [1,1,1,1,1,1,1,1],
                 return (-((.3 + ZEROISH) ** (numOccurences + .1685)) + ONEISH) ** coeffHiddenString;
             }
 
+            function suspiciousElementText(fnode) {
+                const element = fnode.element;
+                var childNode = element.childNodes;
+
+                if(childNode.length > 0) {
+                    return ((checkForSusText(childNode)) ? ONEISH : ZEROISH) ** coeffSusElementText;
+                }
+
+                // if (checkForSusText(element)) {
+                //     var childNode = element.firstChild;
+                //     console.log("Found in first")
+                //     return ((checkForSusText(childNode)) ? ZEROISH : ONEISH) ** coeffSusElementText;
+                // }
+            }
+
+            function checkForSusText(element) {
```

Comment:
This takes a list of elements, not a single one. Furthermore, you're testing only the first element of the list, which will give you trouble if you encounter anything like `<span> <b>Close</b></span>` (with a leading space) or `<a><em>no thanks</em></a>`. If I were you, I'd check innerText on the target element instead of dealing with the child list. Though be aware it can get slow.